Soccar [Physics]/src/nl/soccar/physics/models/WheelPhysics.java

```diff
@@ -172,49 +215,8 @@ public void setDesiredSpeed(ThrottleAction throttleAction) {
      *
      * @param angle the new angle (not relative to the carPhysics) of this Wheel.
      */
-    public void setAngle(float angle) {
-        body.m_sweep.a = carBody.getAngle() + angle;
-    }
-
-    /**
-     * Sets wheel's velocity vector with sideways velocity subtracted.
-     */
-    private Vec2 getLateralVelocity() {
-        Vec2 currentRightNormal = body.getWorldVector(new Vec2(1, 0));
-        return currentRightNormal.mul(Vec2.dot(currentRightNormal, body.getLinearVelocity()));
-    }
-
-    private Vec2 getForwardVelocity() {
-        Vec2 currentRightNormal = body.getWorldVector(new Vec2(0, 1));
-        return currentRightNormal.mul(Vec2.dot(currentRightNormal, body.getLinearVelocity()));
-    }
-
-    @Override
-    public void step() {
-
-        eliminateLateralVelocity();
-
-        if (isSteerable()) {
-            setAngle(carPhysics.getSteerAngle());
-        }
-
-        if (isPowered()) {
-            setDesiredSpeed(carPhysics.getCar().getThrottleAction());
-            updateDrive();
-        }
-
-    }
-
-    @Override
-    public void reset() {
-        setPosition(originalPos.x, originalPos.y, 0, 0, 0, 0);
-    }
-
-    @Override
-    public void setPosition(float x, float y, float degree, float linearVelocityX, float linearVelocityY, float angularVelocity) {
-        body.setLinearVelocity(new Vec2(0.0F, 0.0F));
-        body.setAngularVelocity(0.0F);
-        body.getPosition().set(x, y);
+    private void setAngle(float angle) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "setAngle" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)